### PR TITLE
Update _connections.md

### PR DESCRIPTION
**Pull request by KseniiaNovinskaia**
Adding seats layout endpoint

### CODE REVIEW

File: source/includes/_connections.md [hunk 1]
```diff
@@ -1545,3 +1545,168 @@ Error Code  | Meaning
...
+                }
+            }
+        }
+    ]
+}
+```
+
+This endpoint checks and provides the data on the seats availability for the selected trip. The trip is defined as a ride from one station to another at a particular day and time.
+
+In parallel, we send the outbound request to the operating carrier integration systems to retrieve the latest available seats information.
```
Comment (nwilson-distribusion) [inline source/includes/_connections.md:1682]: I'd remove the "in parallel" here, and I'd change the "systems" to "system."  For this endpoint, we won't be sending multiple requests to multiple carrier systems.

Comment (nwilson-distribusion) [reply source/includes/_connections.md:1682]: Could make the same change on connections/vacancy while we're at it.

File: source/includes/_connections.md [hunk 2]
```diff
@@ -1545,3 +1545,168 @@ Error Code  | Meaning
...
+----------- | -------
+400.000.000 | The provided json is invalid
+400.100.000 | Request invalid
+400.101.004 | Departure station is invalid or missing
+400.101.005 | Arrival station is invalid or missing
+400.101.044 | Departure date is in the past
+400.101.049 | Departure time must be before arrival time
+400.200.001 | API key is invalid or missing
+400.300.006 | Marketing carrier does not support seat layouts
+400.800.040 | Marketing carrier not found
```
Comment (nwilson-distribusion) [inline source/includes/_connections.md:1710]: Can we add our generic 500 error, just for the sake of completeness?

500.100.000 | Service Unavailable

File: source/includes/_connections.md [hunk 3]
```diff
@@ -1545,3 +1545,168 @@ Error Code  | Meaning
...
+                    "y": 4,
+                    "z": 0
+                }
+            }
+        }
+    ]
+}
+```
+
+This endpoint checks and provides the data on the seats availability for the selected trip. The trip is defined as a ride from one station to another at a particular day and time.
```
Comment (nwilson-distribusion) [inline source/includes/_connections.md:1680]: Could we include a note that this is only required for some carriers?  Thinking something like this added at the end of what you have:

> Booking a specific seat may be required for some regions, like Brazil.

As I write that, I'm thinking about exposing when this is required in the API...

Comment (nwilson-distribusion) [reply source/includes/_connections.md:1680]: Honest question, I should know the answer but I don't: how does a retailer pass in a seat number to bookings/create?  Should we add that as well?

[the remark this answers is not in the document]
Comment (KseniiaNovinskaia) [reply source/includes/_connections.md:1680]: I indeed matched it to already existing wording of the api. Agree, it should be changed to the api language. If you don't mind, for now I would keep it like this as I need to do urgent change for Brazil. And in a next few days we would do with Caro full api review when we will adjust all this.

File: source/includes/_connections.md (hunk 1 above)
Comment (KseniiaNovinskaia) [reply source/includes/_connections.md:1682]: Done

File: source/includes/_connections.md (hunk 3 above)
Comment (KseniiaNovinskaia) [reply source/includes/_connections.md:1680]: Indeed, retailer would only know if carrier supports seat layout only by calling this endpoint and either getting an answer or getting an error. But I believe, Jorg had in mind the flag on co/fi, not sure if it was implemented though.

Comment (KseniiaNovinskaia) [reply source/includes/_connections.md:1680]: Done with the change

File: source/includes/_connections.md (hunk 2 above)
Comment (KseniiaNovinskaia) [reply source/includes/_connections.md:1710]: Done

File: source/includes/_connections.md [hunk 4]
```diff
@@ -1677,9 +1677,9 @@ curl -g -X GET \
 }
 ```
 
-This endpoint checks and provides the data on the seats availability for the selected trip. The trip is defined as a ride from one station to another at a particular day and time.
+This endpoint checks and provides the data on the seats availability for the selected trip. The trip is defined as a ride from one station to another at a particular day and time.  Bookings a specific seat may be required for some regions, like Brazil.
```
Comment (nwilson-distribusion) [inline source/includes/_connections.md:1680]: Should change "bookings" to "booking" in last sentence

**Comment by nwilson-distribusion** [comment]
Left one final wording change, but otherwise I think its good to merge.  Thanks Kseniia!

**Pull request merged**